e2e/wdio/test/pageobjects/content-editor.page.js

```diff
@@ -0,0 +1,93 @@
+const Page = require('./page');
+
+class ContentEditorPage extends Page {
+  open() {
+    return super.open('/sitecore/shell/Applications/Content%20Editor.aspx?sc_bw=1');
+  }
+
+  async openExperienceEditor(appName) {
+    this.open();
+    await browser.click({ element: this.getSCNode(appName) });
+    await browser.click({ element: 'span=home' });
+    await this.clickOnPublishTab();
+    const ogWindowHandle = await browser.getWindowHandle();
+    await browser.click({ element: 'span=Experience Editor' });
+    await browser.waitUntil(async function() {
+      return (await browser.getWindowHandles().length) > 1;
+    });
+
+    const windowHandles = await browser.getWindowHandles();
+    const popupWindow = windowHandles.filter((handle) => handle !== ogWindowHandle)[0];
+    await browser.switchToWindow(popupWindow);
+    browser.pause(3000);
+    return ogWindowHandle;
+  }
+
+  async clickOnApp(appName) {
+    await browser.click({ element: `span=${appName}` });
+  }
+
+  async clickOnPublishTab() {
+    await browser.click({ element: 'a=Publish' });
+  }
+
+  getSCNode(itemName) {
+    return `//span[contains(.,"${itemName}")]/../../*[@class="scContentTreeNodeGlyph"]`;
+  }
+
+  async createApiKey() {
```

Comment:
Usually we manage the API key with Sitecore serialization since adding items to Sitecore is not JSS-specific functionality. I can see benefits to doing it through the UI, as this would expose issues with creating items, but this would be testing the Sitecore platform itself, not JSS. Let's discuss with the team.